gloo/benchmark/runner.cc

```diff
@@ -256,15 +302,15 @@ void Runner::printHeader() {
   std::cout << std::setw(11) << ("p50 " + suffix);
   std::cout << std::setw(11) << ("p99 " + suffix);
   std::cout << std::setw(11) << ("max " + suffix);
-  std::cout << std::setw(13) << ("min " + bwSuffix);
-  std::cout << std::setw(13) << ("p50 " + bwSuffix);
-  std::cout << std::setw(13) << ("p99 " + bwSuffix);
-  std::cout << std::setw(13) << ("max " + bwSuffix);
+  std::cout << std::setw(13) << ("avg " + bwSuffix);
   std::cout << std::setw(11) << "samples";
   std::cout << std::endl;
 }
 
-void Runner::printDistribution(int elements, int elemSize) {
+void Runner::printDistribution(
+    int elements,
+    int elementSize,
+    Distribution& samples) {
```

Comment:
const ? It doesn't seem to be modified. Not sure how picky you want to be on const info.